configure

```diff
@@ -516,8 +517,11 @@ elif system == 'darwin':
 
 if 'ARCH' in os.environ.keys():
   march = os.environ['ARCH']
-  report ('Machine architecture set by ARCH environment variable to: ' + march + '\n')
-  cpp_flags += [ '-march='+march ]
+  if march:
```

Comment:
If `ARCH` isn't in `os.environ` dictionary, the `else` branch executes. If it is, but is an empty string, `bool('') -> False`. This is usually idiomatic except for NumPy arrays.